ecommerce/extensions/payment/views/cybersource.py

```
@@ -244,123 +245,156 @@ def _get_basket(self, basket_id):
                 str(error))
             return None
 
-    def validate_notification(self, notification):
+    def get_ids_from_notification(self, notification):
+        transaction_id = notification.get('transaction_id')
+        order_number = notification.get('req_reference_number')
+        try:
+            basket_id = OrderNumberGenerator().basket_id(order_number)
+        except:  # pylint: disable=bare-except
+            logger.error(
```

Should this be `logger.exception` so as to now swallow the stack trace.?

---

Can do.